api/server.js

```diff
@@ -0,0 +1,145 @@
+const express = require('express');
+const helmet = require('helmet');
+const knex = require('knex');
+const knexConfig = require("../knexfile.js");
+
+const db = knex(knexConfig.development);
+ 
+const server = express();
+server.use(express.json());
+server.use(helmet());
+
+
+// GET endpoint
+server.get('/api/zoos', (req, res) => {
+    db('zoos')
+    .then(zoos => {
+      res.status(200).json(zoos)
+    })
+    .catch(error => {
+      res.status(500).json(error)
+    })
+  });
+  
+  // POST endpoint
+  server.post('/api/zoos', (req, res) => {
```

Comment:
you want another catch here (in an `if` statement) to make sure that `req.body` has a `name` property - if not, you don't want to accept it. Right now, it doesn't accept it, but it goes to your 500 error and it doesn't help the user understand why it wasn't accepted